src/utils/misc.ts

```diff
@@ -1,13 +1,12 @@
 import * as child_process from 'child_process';
 import { promises as fs } from 'fs';
-import { URL } from 'url';
 import { promisify } from 'util';
 import { TextDocument } from 'vscode-languageserver-textdocument';
 import { Range } from 'vscode-languageserver-types';
 import * as path from 'path';
 
 export async function fileExists(fileUri: string): Promise<boolean> {
```

Comment:
Please check if the calling function  passes `fileUri` or `filePath`

Reply:
If it is file path the `fileUri` parameter can be renamed to `filepath` to avoid confusion

Reply:
It is passing fileUri as a string